runners/spark/src/test/java/org/apache/beam/runners/spark/GlobalWatermarkHolderTest.java

```diff
@@ -65,17 +65,17 @@ public void testLowHighWatermarksAdvance() {
             instant.plus(Duration.millis(5)),
             instant.plus(Duration.millis(5)),
             instant));
-    GlobalWatermarkHolder.advance(jsc);
```

Comment:
the fact that `JavaSparkContext` is not needed here at all anymore makes me happy 😄